capstone/src/main/java/com/google/sps/gmail/GmailAPILoader.java

```diff
@@ -0,0 +1,98 @@
+package com.google.sps.gmail;
+
+import com.google.api.client.auth.oauth2.Credential;
+import com.google.api.client.googleapis.auth.oauth2.GoogleCredential;
+import com.google.api.client.googleapis.javanet.GoogleNetHttpTransport;
+import com.google.api.client.http.javanet.NetHttpTransport;
+import com.google.api.client.json.JsonFactory;
+import com.google.api.client.json.jackson2.JacksonFactory;
+import com.google.api.services.gmail.Gmail;
+import com.google.common.collect.Streams;
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.net.HttpURLConnection;
+import java.net.URL;
+import java.net.URLEncoder;
+import java.security.GeneralSecurityException;
+import java.util.LinkedHashMap;
+import java.util.Map;
+import java.util.stream.Collectors;
+import org.json.JSONObject;
+
+public class GmailAPILoader {
+  private static final JsonFactory JSON_FACTORY = JacksonFactory.getDefaultInstance();
+  private static final String APPLICATION_NAME = "clubhub-step-2020";
```

Comment:
I think this will be shared with Calendar, so maybe put this in constants?

Reply:
Just added both application_name and json_factory to the Constants.java file! Will push in the next commit